defi/src/protocols/data2.ts

```diff
@@ -9931,7 +9931,7 @@ const data2: Protocol[] = [
     cmcId: "9423",
     category: "Indexes",
     chains: ["Ethereum", "Avalanche"],
-    oracles: ["Chainlink", "TWAP"],
+    oracles: ["API3","Chainlink", "TWAP"],
```

Comment:
and this one?

Reply:
ah forgot about this one:
docs: https://docs.phuture.finance/protocol-1/about

under "Broader Asset pricing"